tests/console/oprofile.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Simple tests for oprofile.
+# - profile example application (fio to create some load)
+# - check report
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use strict;
+use warnings;
+use base 'consoletest';
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    # Install fio as load generator and oprofile
+    zypper_call "in fio oprofile";
+
+    # Start a system wide profiling
+    type_string "operf --system-wide &";
+
+    # Start load and stop profiler when finished
+    assert_script_run "fio --name=randwrite --filename=/tmp/randwrite --bs=4k --size=1G; killall -s SIGINT operf";
+
+    # Make sure the report is generated correctly and also check for an entry for the fio load
+    assert_script_run "opreport";
+    assert_script_run "opreport | grep 'fio'";
```

Comment:
By the way do you check that the `operf` running on background finished?

Reply:
See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10656#discussion_r454928180 
If you guys say that it is safer to add a `script_run('kill %1');` just in case, I will add it!